src/classification.jl

```diff
@@ -107,13 +107,13 @@ immutable LabelMap{K}
     vs::Vector{K}
     v2i::Dict{K,Int}
 
-    function LabelMap(vs, v2i)
+    function LabelMap{K}(vs, v2i) where K
```

Comment:
Ah, I missed this on the earlier pass. This is also 0.6-only. At this point I think we can just drop support for earlier versions of Julia and update the REQUIRE to `julia 0.6`.